src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+import java.io.File;
+
+public class Main {
+
+    public static void main(String[] args) {
+        if (args.length == 2 && args[0].equals("-fileName")) {
+            ReadManager readManager = new ReadManager(new File(args[1]));
+            SearchEngine searchEngine = new SearchEngine(readManager);
+            searchEngine.run();
+        } else {
+            throw new IllegalArgumentException("Enter the '-fileName' and the path to the document.txt");
```

Comment:
Is it's a file name or file path? The file is called `document.txt` or `text.txt`?

Reply:
renamed to filePath and change abstract "document.txt" to real "text.txt"